lib/rucio/web/ui/flask/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright European Organization for Nuclear Research (CERN)
```

Comment:
why is this copyright different than the others?

Reply:
No idea, didn't even look at it. Just copied it from the main ui folder.